pipeline-templates/stage/build.yml

```diff
@@ -0,0 +1,63 @@
+stages:
+- stage: build
+  displayName: Build
+  jobs:
+  - job: build_image
+    displayName: Build
+    pool:
+      name: DAS - Continuous Integration
```

Comment:
Not part of your change but could this have a demands - docker like das-self-service? The build fails when it selects windows pod

Reply:
Added demand